index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment:
```suggestion
<div class="game-over-message" hidden>Game over, refresh to play again 🧑‍💻 🤖!</div>

```
This would make the code more meaningful to screen readers and other assistive technologies, as well as search engines.